src/meter/Meter.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import { props, t, skinnable } from '../utils';
+import cx from 'classnames';
+import FlexView from '../flex/FlexView';
+import find from 'lodash/collection/find';
+import { warn } from '../utils/log';
+
+const Range = t.refinement(t.struct({
+  startValue: t.maybe(t.Number),
+  endValue: t.Number,
+  fillingColor: t.String,
+  labelColor: t.String
+}), r => r.startValue < r.endValue);
```

Comment:
protcombtip: always give custom types a name (last parameter of any combinator is an optional string `name`). It makes debugging less painful